### DISCUSSION TITLE
Method not found: 'Void Avalonia.Markup.Xaml.RuntimeXamlDiagnostic.set_Document(System.String)'

### DISCUSSION THREAD

**Discussion by pyking-wang** [AvaloniaUI/Avalonia#13666] [Q&A]
I upgraded from version '11.0.999-cibuild0039056-beta' to version '11.1.999-cibuild0042021-beta', 
but the original project showed this error message and the previewer was broken. 
But the program is running normally. 
Does anyone know what the reason is? Thank you
![image](https://github.com/AvaloniaUI/Avalonia/assets/76192669/21b348b6-7e09-4833-b657-54ff9b5b1e38)

**Comment by maxkatz6** [comment]
Double-check if you don't have mixed versions in the package.

**Reply by pyking-wang** [reply]
![image](https://github.com/AvaloniaUI/Avalonia/assets/76192669/8a1ab993-0bb1-4d86-ae66-1e085cb99494)
It's all the latest.

**Reply by timunie** [reply]
also all your other dependencies may be causing this. Does it happen in a blank project?

**Reply by pyking-wang** [reply]
No, I think it may be due to third-party libraries referenced in the project. I have temporarily rolled back the version and will test it when I have time.

**Reply by timunie** [reply]
That's what I meant as well. Any third-party lib targeting another version of Avalonia ;-)
